daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -908,21 +911,36 @@ object Main extends Logging {
                   } else {
                     // not streaming, show left over data warning
                     val Dump = new DataDumper
-                    val curPositionInByte0b = inStream.inputSource.position
+                    val firstByteBitArray = new Array[Byte](1)
+                    val bitsAlreadyConsumed = loc.bitPos0b % 8
+                    val firstByteString = if (bitsAlreadyConsumed != 0) {
+                      val bitsToDisplay = 8 - bitsAlreadyConsumed
+                      val pbp = inStream.inputSource.position + 1
+                      inStream.fillByteArray(firstByteBitArray, bitsToDisplay, finfo)
```

Comment:
Is it possible to use ``inStream.getByteArray()`` instead of making fillByteArray public? It's probably not a big deal if that's made public, but ideally it should only be used internally since it doesn't do any sanity checking like other functions do.

Reply:
Yes, I missed that function during the first go round. I've made the updates!